64_FFT/app_resources/common.hlsl

```diff
@@ -0,0 +1,15 @@
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+using scalar_t = nbl::hlsl::float32_t;
+
+struct PushConstantData
+{
+	uint64_t inputAddress;
+	uint64_t outputAddress;
+	uint32_t dataElementCount;
+};
+
+#define _NBL_HLSL_WORKGROUP_SIZE_ 64
```

Comment:
take the workgroup size as a parameter into your `workgroup::FFT` template, just how the scans do

Reply:
Done! Ended up in [this PR](https://github.com/Devsh-Graphics-Programming/Nabla/pull/734)